service/controller/key/key.go

```diff
@@ -18,9 +18,10 @@ const (
 	// Namespace is the namespace where App CRs are created.
 	Namespace = "giantswarm"
 
-	LabelAppOperatorVersion = "app-operator.giantswarm.io/version"
-	LabelManagedBy          = "giantswarm.io/managed-by"
-	LabelServiceType        = "giantswarm.io/service-type"
+	LabelAppOperatorVersion    = "app-operator.giantswarm.io/version"
+	LabelConfigOperatorVersion = "config-controller.giantswarm.io/version"
```

Comment:
This annotation is available in apiextensions. IMO it should be imported from there.